apptools/apptools-android-tests/apptools/build.py

```diff
@@ -58,7 +58,10 @@ def test_build_release(self):
 
     def test_build_missing_so_file(self):
         comm.setUp()
-        comm.create(self)
+        cmd = comm.HOST_PREFIX + comm.PackTools + \
+            "crosswalk-app create org.xwalk.test --android-crosswalk=" + \
```

Comment:
Due to the shared mode apk hasn't libxwalkcore.so file, so it isn't need to test.